tmk_core/common/keyboard.c

```diff
@@ -274,7 +283,7 @@ void keyboard_task(void)
     matrix_scan();
 #endif
 
-    if (is_keyboard_master()) {
+    //if (is_keyboard_master()) {
```

Comment:
This has bugged me for a while ....

I'm not sure how the corne code handles it without disabling it, but that may be a better solution than this. 

But in the meanwhile, it may be a good idea to have this as a configurable option.  Eg 
```suggestion
    #ifndef SPLIT_KEYBOARD_MIRRORING
    if (is_keyboard_master()) 
    #endif
    {
```

Reply:
The corne basically calls straight into what that calls bypassing it if I remember correctly.
That or is_keyboard_master() just always returns true as it's not implemented on the corne's split code.

Reply:
Will add this once testing is complete